network-api/networkapi/wagtailpages/pagemodels/products.py

```diff
@@ -636,6 +636,14 @@ def creepiness(self):
             average = 50
         return average
 
+    @property
```

Comment:
This is used to return the key of the most voted rating. We then use this on the front end to decide what face icon to show